README.rst

```diff
@@ -90,12 +90,20 @@ For humidity compensated raw gas readings, we'll need a secondary sensor such as
     while True:
         temperature = bme280.temperature
         humidity = bme280.relative_humidity
-        compensated_raw_gas = sgp.measure_raw(temperature = temperature, relative_humidity = humidity)
+
+        # For compensated raw gas readings
+        compensated_raw_gas = sgp.raw(temperature = temperature, relative_humidity = humidity)
+
+        # For Compensated voc index readings
```

Comment:
There's might also need to be a time.sleep(1) here, or we can comment out one of the examples (in this case `compensated_raw_gas` would be better to comment out) because I think the sensor needs full second between readings, and both the `measure_raw` and `measure_index` make a call to the sensor over i2c.

Reply:
Nice catch, datasheet says 1s typical, .5s min